shell/src/main/java/tachyon/shell/command/RmCommand.java

```diff
@@ -44,16 +49,38 @@ public String getCommandName() {
   }
 
   @Override
-  void runCommand(TachyonURI path) throws IOException {
+  protected int getNumOfArgs() {
+    return 1;
+  }
+
+  @Override
+  protected Options getOptions() {
```

Comment:
We could save a bunch of boilerplate by defining the recursive option as a protected constant in `AbstractTfsShellCommand`, and having this entire function body be 
`return new Options().addOption(RECURSIVE_OPTION)`